nilearn/datasets/func.py

```diff
@@ -206,6 +143,13 @@ def fetch_haxby(data_dir=None, n_subjects=1, fetch_stimuli=False,
             **kwargs)
 
 
+@deprecated("fetch_haxby_simple will be removed in future releases. "
+            "Calling 'fetch_haxby' instead.")
```

Comment:
I am a bit worried that the signatures of fetch_haxby and fetch_haxby_simple are different (both input signature and return values).

We should probably keep the old fetch_haxby_simple function for a little while, but flag it as deprecated, and remove it from the documentation.